client/mass/store.js

```diff
@@ -459,6 +459,7 @@ function validatePlot(p, vocabApi) {
 		} else if (p.chartType == 'Disco') {
 		} else if (p.chartType == 'profileBarchart') {
 		} else if (p.chartType == 'profilePolar' || p.chartType == 'polar') {
+		} else if (p.chartType == 'sampleView') {
```

Comment:
later can improve for validatePlot() to import an optional validator from the chart script and run it here